README.md

```diff
@@ -48,4 +48,29 @@ IOs are lazy so they don't actually do anything until they are interpreted.
 `promisedResult` will return a promise of a FiberResult.
 
 ## Concurrency Abstractions
-Waveguide also provides Ref (synchronous mutable cell), Deferred (set once asynchronous cell), Semaphore, and an asynchronous Queue implementation.
+Waveguide also provides Ref (synchronous mutable cell), Deferred (set once asynchronous cell), Semaphore, and an asynchronous queue implementation.
+
+## Compared to...
+
+### funfix
+[funfix](https://github.com/funfix/funfix/) provides a whole suite of functional tools in its core such as either, option, and others.
+In contrast, waveguide has a peer dependency on [fp-ts](https://github.com/gcanti/fp-ts/) for this machinery and exports instances for fp-ts typeclasses where appropriate.
+
+Waveguide does not attempt to provide an equivalent to funfix's weaker primitives such as Eval or Future. 
```

Comment:
'weaker' in what sense?

Reply:
funfix's provides several effect monads of which IO is the most powerful. Eval does not model asynchronous boundaries. Future does not model errors. With IO from waveguide you just get all the capabilities.

Reply:
@rzeigler Thanks! I know those details from using funfix but I think it's good to express that meaning in the doc as `weaker` it can be misleading.